plugins/extend-give-wp/src/admin/admin-page.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ *  The Extend GiveWP plugin option settings page.
+ *
+ * @package    spiralWebDB\ExtendGiveWP\Admin
+ *
+ * @since      1.0.0
+ *
+ * @author     Robert A. Gadon
+ *
+ * @link       http://spiralwebdb.com
+ *
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDB\ExtendGiveWP\Admin;
+
+use function spiralWebDB\ExtendGiveWP\_get_plugin_dir;
+
+add_action( 'admin_menu', __NAMESPACE__ . '\add_option_settings_page' );
+/*
+ * Add an option settings page to the plugin admin.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function add_option_settings_page() {
+	$hookname = add_submenu_page(
+		'options-general.php',  // parent slug
+		'Extend GiveWP -- Donation Form Option Settings', // page title
+		'Extend GiveWP', // menu title
+		'manage_options', // capability
```

Comment:
Change to `'manage_categories'` to allow Editors access.

Reply:
Done in commit https://github.com/rgadon107/cornerstone/pull/52/commits/e12a3dbceede0b58d31b3d392d0e0a068a4b0315